includes/classes/class-rtcamp-google-embeds.php

```diff
@@ -216,11 +225,20 @@ public function wpdocs_embed_handler_google_drive( $matches, $attr, $url ) {
 			return '';
 		}
 
+		$file_id = '';
+		$basedir = '';
+		$baseurl = '';
+
+		$file_id = $this->get_file_id_from_url( $url );
+		$cached_data = $this->get_cached_data_from_file_id( $file_id );
```

Comment:
We're already getting cached file URL in `get_thumbnail_url` function, then why do we need this code again? `$thumbnail_url` itself is a cached URL here if the file is cached.